src/bbmap_bbsplit/config.vsh.yaml

```diff
@@ -0,0 +1,179 @@
+namespace: "bbmap"
+name: "bbmap_bbsplit"
+description: Split sequencing reads by mapping them to multiple references simultaneously.
+links:
+  homepage: https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/
+  documentation: https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbmap-guide/
+  repository: https://github.com/BioInfoTools/BBMap/blob/master/sh/bbsplit.sh
+
+license: BBTools Copyright (c) 2014
+
+argument_groups:
+- name: "Input"
+  arguments:
+  - name: "--id"
+    type: string
+    description: Sample ID
+  - name: "--paired"
+    type: boolean_true
+    description: Paired fastq files or not?
+  - name: "--input"
+    type: file
+    multiple: true
+    description: Input fastq files, either one or two (paired), separated by ";".
+    multiple_sep: ","
+    example: reads.fastq
+  - name: "--primary_ref"
+    type: file
+    description: Primary reference FASTA
+  - name: "--other_ref_names"
+    type: file
+    description: |
+      Path to comma-separated file containing a list of reference genomes to filter reads
+      against with BBSplit.
```

Comment:
Can you please comment on the design of this interface? How does it relate the way bbsplit itself handles references?

Reply:
Indeed it is different from the way bbsplit works, it's because I was trying to keep this version similar to the rnaseq version (which i believe is partly based on the nf-core version but I could be wrong). I just kept the same structure and shortened some on the argument names, like the --index argument mentioned in your comment below. 

Another thing I just realized is that most of the bbmap arguments can be used with bbsplit as well as they mention in the help message. Should I add these as well? There are a lot so I would probably need to rework the whole script in that case(which I don't mind doing of course).

Reply:
I suggest we stick to the [biobox way of working](https://github.com/viash-hub/biobox/blob/main/CONTRIBUTING.md#step-7-add-arguments-for-the-input-files) as much as possible. That means we use (where possible) the original arguments of the tool.

We will have to update the RNAseq module call, but that's not an issue.

Reply:
> Another thing I just realized is that most of the bbmap arguments can be used with bbsplit as well as they mention in the help message. Should I add these as well? There are a lot so I would probably need to rework the whole script in that case(which I don't mind doing of course).

`bbmap` and `bbsplit` for me are two separate tools, but I may be wrong. What do you think would be the benefit of adding those arguments?

Reply:
Bbsplit uses bbmap internally so the same options are available as when you call bbmap directly, from what I understand, so they are not exactly separate tools.